_posts/2023-06-23-podman-discovery.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: Discovering JVMs in Podman
+date: 2023-06-23
+synopsis: Explore Cryostat 2.3's new Podman discovery mechanism
+author: Andrew Azores
+---
+
+#### Table of Contents
+* auto-gen TOC:
+{:toc}
+<hr>
+
+## Introduction
+
+Last month, we [released Cryostat 2.3.0](/2023/05/26/cryostat-release) and called out `Podman API Discovery` as a notable new feature. Let's take a quick dive into that.
+
+## Background
+
+Older versions of Cryostat supported automatic startup detection of one deployment platform at a time (Kubernetes API or JDP), plus the Custom Targets system. In Cryostat 2.2 a new Discovery Plugin system was introduced, enabling additional mechanisms for target application detection alongside those mentioned above. In Cryostat 2.3 this is expanded once again with the Cryostat Agent - more details on that in another post to come - which implements a Discovery Plugin, with a reorganized implementation that allows multiple deployment platforms to be enabled at once, and with a brand new deployment platform implementation for the Podman REST API.
```

Comment:
Some commas should be added after `In Cryostat 2.3`, twice. 

Everything else looks good!

Reply:
probably same for:

> In Cryostat 2.2 a new Discovery